### DISCUSSION TITLE
Blank white screen, won't display react app

### DISCUSSION THREAD

**Discussion by jmrocco** [community/community#23595] [Other Feature Feedback, Questions, & Ideas]
<p>My repo: <a href="https://github.com/jmrocco/personal_website" class="inline-onebox" rel="noopener nofollow ugc">GitHub - jmrocco/personal_website: Version 2 of my personal website.</a><br>
My website: <a href="https://julietterocco.com/" rel="noopener nofollow ugc">https://julietterocco.com/</a></p>
<p>For the life of me I can not figure out why my github pages can’t load any of my source files. It only loads index.html which is a white blank screen. I’ve seen a few similar posts about this issue but have not found any common discrepancies.</p>
<p>I have installed through npm the gh-pages module and done “npm run deploy”; this created my gh-pages branch. I’ve updated my package.json with the appropriate information. As far as I can tell I have no capital letters in my repository…</p>
<p>I hosted another website previously on Github Pages at this domain with no issues. It appears that Github pages and the process has changed since then and I can’t get it to work.</p>

**Comment by logankilpatrick** [comment]
<p>Hey, It looks like things are working now (awesome personal site btw!). What ended up resolving the issue?</p>

**Comment by jmrocco** [comment, marked as answer]
<p>Thank you! In the guides I had read online about setting up Github Pages, they said to add the homepage property to <code>package.json</code> as the following <code>https://{username}.github.io/{repo-name}</code> . After receiving some help, I had to change it to <code>https://{username}.github.io/</code> . The repository name was causing the site to reroute to something that doesn’t exist.</p>

**Reply by lokeshahire** [reply]
Thank You.. After Spending 3 hrs , finally find a exact solution

**Reply by mekdie** [reply]
Do you have to npm run deploy again? If I want to update something into the contents, do I need to delete the branch and redeploy it? I'm still new and learning react so not sure how it works haha

**Reply by miketheceleb** [reply]
I'm having this same issue. Haven't figured it out yet. Feel free to look at my repo miketheceleb.github.io/chitchatreact
//this is my practice project :-)

**Reply by CpE-Duane** [reply]
Tried to do that but its still a blank page, check mine https://cpe-duane.github.io/multi-form/

**Reply by awehstino** [reply]
I'm having same problem can't fix it

**Reply by awehstino** [reply]
Feel like my brain is leaking

**Reply by Lecon-a** [reply]
> Tried to do that but its still a blank page, check mine https://cpe-duane.github.io/multi-form/

I am experiencing similar thing now

**Reply by awehstino** [reply]
I've solved the problem

**Reply by awehstino** [reply]
> > Tried to do that but its still a blank page, check mine https://cpe-duane.github.io/multi-form/
> 
> I am experiencing similar thing now

Screenshot your json file and send it to Augusthne123@gmail.com let me see

**Reply by awehstino** [reply]
> Tried to do that but its still a blank page, check mine https://cpe-duane.github.io/multi-form/

Open your Json file screenshot and send it to Augusthne123@gmail.com let me see

**Reply by NavarroErk** [reply]
> I've solved the problem

how did you manage to solve the problem? im still having that issue at this moment

**Reply by Dugicha** [reply]
guys did u solve this problem i tried only adding  https://{username}.github.io/ to json "homepage" but still does not work
![image](https://user-images.githubusercontent.com/78078192/236181270-0fe4df64-f188-437a-9b1b-348282211a92.png)

**Reply by ayotomiwasalau** [reply]
Same here, still having the issue

**Reply by Nolanjue** [reply]
set private = 'false' under homepage and name

**Reply by awehstino** [reply]
> guys did u solve this problem i tried only adding  https://{username}.github.io/ to json "homepage" but still does not work
> ![image](https://user-images.githubusercontent.com/78078192/236181270-0fe4df64-f188-437a-9b1b-348282211a92.png)
> 

in your "script" : {
        >>>>>>   "start" : "react-script start"
}  you dont have it

**Reply by prashus23** [reply]
https://github.com/prashus23/prashus23.github.io/blob/main/package.json


i m facing the same issue done all the thing to private to false please anyone can help me with this same issue that will be very helpfull.

**Comment by logankilpatrick** [comment]
<p>Great, glad things are working now. Happy holidays!</p>

**Comment by RachaelSMathew** [comment]
<p>Hi,<br>
I am having the same issue as jmrocco was having here. I’ve tried both https://{username}.github.io/{repo-name} and https://{username}.github.io for the homepage in package.json and I’m still getting a blank white screen when I’ve deployed.</p>

**Reply by Shady-ops** [reply]
Go to package.json, find private proprety true and make it false

**Reply by miketheceleb** [reply]
I did this and still having issues..

**Reply by CpE-Duane** [reply]
Hi facing same issue now. Did you guys solve it???

**Comment by jfullstackdev** [comment]
<p>GitHub Pages is already hosting the site but the problem is that the links are wrong, for example,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/b/fb5c35b30acbf2106d44f7b3c5a578706e63b1e4.png" data-download-href="/uploads/short-url/zRDp1oCIGVXFG4wKyZQ7uFpimfa.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181078128-a1ac62a8-f139-4dc8-91e7-6334a07479c4.png" alt="image" data-base62-sha1="zRDp1oCIGVXFG4wKyZQ7uFpimfa" width="690" height="251" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/b/fb5c35b30acbf2106d44f7b3c5a578706e63b1e4_2_690x251.png, https://aws1.discourse-cdn.com/github/optimized/3X/f/b/fb5c35b30acbf2106d44f7b3c5a578706e63b1e4_2_1035x376.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/f/b/fb5c35b30acbf2106d44f7b3c5a578706e63b1e4.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/b/fb5c35b30acbf2106d44f7b3c5a578706e63b1e4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1249×456 14.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>there is no such thing as <code>/react-gh-pages/static/js/main.7e2b7cdf.js</code>, try to change all <code>react-gh-pages</code> to <code>rachael-site</code> because this is the name of the repo,</p>
<p>for example,</p>
<p><code>/rachael-site/static/js/main.7e2b7cdf.js</code></p>
<p><a href="https://rachaelsmathew.github.io/rachael-site/static/js/main.7e2b7cdf.js">https://rachaelsmathew.github.io/rachael-site/static/js/main.7e2b7cdf.js</a><br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/1/0124d040ec2f84155ee1e6a24b22b076cea49b02.png" data-download-href="/uploads/short-url/a7lHrh7pK7vqiUPpxFMZ3jd7Oy.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181078129-6a20e81b-a08a-4229-96d4-61ad08b14a09.png" alt="image" data-base62-sha1="a7lHrh7pK7vqiUPpxFMZ3jd7Oy" width="690" height="276" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/1/0124d040ec2f84155ee1e6a24b22b076cea49b02_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1164×466 59.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>this is loading your JavaScript,</p>

**Comment by CpE-Duane** [comment]
check your default route, if the landing page of your website and your
default route is same

**Comment by CpE-Duane** [comment]
Check your router if it has / path

**Comment by anessheremeti** [comment]
{
    "homepage": "https://Anes039.github.io/",
    "name": "third-task",
    "version": "0.1.0",
    "private": false,
    "dependencies": {
        "@testing-library/jest-dom": "^5.17.0",
        "@testing-library/react": "^13.4.0",
        "@testing-library/user-event": "^13.5.0",
        "animate.css": "^4.1.1",
        "react": "^18.2.0",
        "react-dom": "^18.2.0",
        "react-scripts": "5.0.1",
        "react-spinners": "^0.13.8",
        "web-vitals": "^2.1.4"
    },
    "scripts": {
        "start": "react-scripts start",
        "predeploy": "npm run build",
        "deploy": "gh-pages -d build",
        "build": "react-scripts build",
        "test": "react-scripts test",
        "eject": "react-scripts eject"
    },
    "eslintConfig": {
        "extends": [
            "react-app",
            "react-app/jest"
        ]
    },
    "browserslist": {
        "production": [
            ">0.2%",
            "not dead",
            "not op_mini all"
        ],
        "development": [
            "last 1 chrome version",
            "last 1 firefox version",
            "last 1 safari version"
        ]
    },
    "devDependencies": {
        "gh-pages": "^6.0.0"
    }
}
Here i have the same issue i tried to change the path and private property to false but again facing the same issues.....

**Comment by awehstino** [comment]
and also go to your repo in github under settings SELECT pages and unpublished the site then go make the changes add, commit, push then run your command to deploy your website if successful it will show site publishedkeep me updateSent from my Huawei phone-------- Original message --------From: Anes Sheremeti ***@***.***>Date: Fri, 29 Sep 2023, 9:35 amTo: community/community ***@***.***>Cc: Black fugee ***@***.***>, Comment ***@***.***>Subject: Re: [community/community] Blank white screen, won't display react app (Discussion #23595)
{
"homepage": "https://Anes039.github.io/",
"name": "third-task",
"version": "0.1.0",
"private": false,
"dependencies": {
***@***.***/jest-dom": "^5.17.0",
***@***.***/react": "^13.4.0",
***@***.***/user-event": "^13.5.0",
"animate.css": "^4.1.1",
"react": "^18.2.0",
"react-dom": "^18.2.0",
"react-scripts": "5.0.1",
"react-spinners": "^0.13.8",
"web-vitals": "^2.1.4"
},
"scripts": {
"start": "react-scripts start",
"predeploy": "npm run build",
"deploy": "gh-pages -d build",
"build": "react-scripts build",
"test": "react-scripts test",
"eject": "react-scripts eject"
},
"eslintConfig": {
"extends": [
"react-app",
"react-app/jest"
]
},
"browserslist": {
"production": [
">0.2%",
"not dead",
"not op_mini all"
],
"development": [
"last 1 chrome version",
"last 1 firefox version",
"last 1 safari version"
]
},
"devDependencies": {
"gh-pages": "^6.0.0"
}
}
Here i have the same issue i tried to change the path and private property to false but again facing the same issues.....

**Reply by anessheremeti** [reply]
T https://anes039.github.io/static/js/main.973803ce.js net::ERR_ABORTED 404 (Not Found)
anes039.github.io/:1     GET https://anes039.github.io/static/css/main.21cf0024.css net::ERR_ABORTED 404 (Not Found)
favicon.ico:1     GET https://anes039.github.io/favicon.ico 404 (Not Found)
Image (async)
(anonymous) @ inpage.js:1
u @ inpage.js:1
a @ inpage.js:1
(anonymous) @ inpage.js:1
r.sendSiteMetadata @ inpage.js:1
e @ inpage.js:1
/manifest.json:1     GET https://anes039.github.io/manifest.json 404 (Not 
here are the errors now even in github is not working
